core/run_ctest/README.md

```diff
@@ -24,7 +25,7 @@ p_input = {
 ```
 "maven_args": ["-DfailIfNoTests=false"]
 ```
-
+*For "ctest_selected", please specify the ctests you want to run in the input file, if you want to run all the ctests, just remain the file empty. 
```

Comment:
I don't really like this design. Can we simply pass the test name as a parameter through the command line? It's kinda awkward to open a file and put the name there and then call the command (too many unneceesary keystroke)..

Reply:
Ok